waku/v2/waku_archive/driver/postgres_driver/asyncpool.nim

```diff
@@ -126,11 +126,12 @@ proc releaseConn(pool: PgAsyncPool, conn: DbConn) =
     if pool.conns[i].dbConn == conn:
       pool.conns[i].busy = false
 
-proc query*(pool: PgAsyncPool,
-            query: string,
-            args: seq[string] = newSeq[string](0)):
-            Future[Result[seq[Row], string]] {.async.} =
+proc runQuery*(pool: PgAsyncPool,
```

Comment:
why the rename?

Reply:
hehe, the reason was to avoid the next :)

`let rowsRes = await s.connPool.query(query, args)`
and instead have:
`let rowsRes = await s.connPool.runQuery(query, args)`

Reply:
Hmm..no strong feelings TBH:)

But I am a sucker for consistency, so I checked the SQLite implementation: https://github.com/waku-org/nwaku/blob/master/waku/common/sqlite/database.nim#L223

Reply:
Good point! I'll rename it back to keep it consistent. With less priority though, I'll submit a separate PR to try to enhance that proc name.
Thanks!